src/wfs/src/test/java/org/geoserver/wfs/v2_0/MultiDimensionTest.java

```diff
@@ -0,0 +1,92 @@
+package org.geoserver.wfs.v2_0;
```

Comment:
Copyright header missing ;-)

Reply:
shouldn't that be a QA fail, I actually checked that today!

Reply:
I should... I think that I gave up doing that when I set up QA, because too many existing tests had copyright headers missing.